miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultWorkstationService.java

```diff
@@ -92,4 +95,31 @@ public ValidationResult validateDeletion(Workstation object) throws IOException
     return result;
   }
 
+  @Override
+  public long update(Workstation workstation) throws IOException {
+    authorizationManager.throwIfNonAdmin();
+    Workstation newWorkstation = get(workstation.getId());
+    validateChange(workstation, newWorkstation);
+    applyChanges(newWorkstation, workstation);
+    return save(newWorkstation);
+  }
```

Comment:
Override the `validateChange` and `authorizeUpdate` instead of `update` - the abstract superclass implements the standard update procedure, and there's no reason to modify that here